fineract-provider/src/main/resources/sql/migrations/core_db/V312__address_module_tables_metadat.sql

```diff
@@ -0,0 +1,227 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one
+-- or more contributor license agreements. See the NOTICE file
+-- distributed with this work for additional information
+-- regarding copyright ownership. The ASF licenses this file
+-- to you under the Apache License, Version 2.0 (the
+-- "License"); you may not use this file except in compliance
+-- with the License. You may obtain a copy of the License at
+--
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing,
+-- software distributed under the License is distributed on an
+-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+-- KIND, either express or implied. See the License for the
+-- specific language governing permissions and limitations
+-- under the License.
+
+
+
+-- code inserts
+INSERT INTO `m_code` (`code_name`, `is_system_defined`) VALUES ('STATE',1);
+INSERT INTO `m_code` (`code_name`, `is_system_defined`) VALUES ('COUNTRY',1);
+INSERT INTO `m_code` (`code_name`, `is_system_defined`) VALUES ('ADDRESS_TYPE',1);
+
+-- inserts for STATE
+INSERT INTO `m_code_value` (`code_id`, `code_value`,`order_position`, `is_active`) 
+VALUES(
+ (select id from m_code where code_name = 'STATE'), 'Andhra Pradesh', 0,1);
```

Comment:
I don't think we need to add states of India as part of SQL migration. This is admin job and he will be adding countries and states from the UI.